source/wp-content/themes/wporg-developer-2023/src/code-short-title/style.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable scss/no-global-function-names */
```

Comment:
SCSS lint says to `@use sass:color` and `color.adjust()` rather than `darken()` but that doesn't compile, I assume because our sass version is misaligned with the linter.